app/src/androidTest/java/ch/epfl/sdp/peakar/social/SocialActivityTest.java

```diff
@@ -71,6 +137,27 @@ public void TestTopBarColor() {
         onView(ViewMatchers.withId(R.id.top_bar)).check(matches(withBackgroundColor(R.color.LightGrey)));
     }
 
+    /* Test that the items in the global rankings are sorted correctly */
+    @Test
+    public void TestSortingInGlobalRankings() {
+        try {
+            Thread.sleep(LONG_SLEEP_TIME);
```

Comment:
Is there a reason why a sleep was added at the beginning of the test?

Reply:
Yes. As the rankings loading is not an immediate operation, it is necessary to wait for it.